attestationreport/attestationreport.go

```diff
@@ -1234,7 +1215,7 @@ func (hws *hwSigner) SignPayload(payload []byte, alg jose.SignatureAlgorithm) ([
 		ret := make([]byte, 2*keySize)
 		_, err = asn1.Unmarshal(asn1Sig, &esig)
 		if err != nil {
-			return nil, errors.New("ECDSA Signature was not in expected format.")
```

Comment:
I would recommend keeping `ECDSA` and adding it as a known initialism: https://staticcheck.io/docs/configuration/options/#initialisms

Reply:
@dominikh Thanks for the feedback! Included the capitalization again and it works fine. Not sure what went wrong there previously, guess I was just overeager removing the capitalizations. Will push it with the next PR